playbook/app/pb_kits/playbook/pb_date_picker/date_picker_helper.js

```diff
@@ -166,6 +166,9 @@ const datePickerHelper = (config) => {
     picker.input.style.caretColor = 'transparent'
     picker.input.style.cursor = 'pointer'
   }
+
+  // Fix event bubbling bug on wrapper
```

Comment:
Stupid I know but could you change this to "fixes"? I read it like a "Fix this" note.